mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/GhostMailbox.java

```diff
@@ -0,0 +1,31 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.mailbox.cassandra;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public interface GhostMailbox {
```

Comment:
it's not really an interface : it defines no method.
You could have a class GhostMailboxLogger that would have a logger method calling `MDCStructuredLogger.forLogger(GhostMailbox.LOGGER)`, WDYT ?

Reply:
If you think it's better...

Reply:
ghostmailbox is a funny name, but we are the only team in the world to know what it means. So either you change the name, or you add some javadoc to explain the naming (or a reference to the ticket you open)